### PR TITLE
Add and update new, improved widgets & aggregations 

### PR DESCRIPTION
**Pull request by dulanism**
The following is revised or new

- [x] Update to Aggregation builder options, e.g. Group By
- [x] New Log View Chapter
- [x] Exporting option for Enterprise
- [x] Widget focus

### CODE REVIEW

**Comment by dulanism** [comment]
> @bernd Mind checking this out to see what's going on? I'm getting a 404 on a new image. Also, verify if the index structure is correct. In this one, I'm adding a new Log View page in the Enterprise section.

@dulanism The "toctree contains reference to nonexisting document 'pages/enterprise/log_view'" means, that there is no `pages/enterprise/log_view.rst` file. You reference `pages/enterprise/log_view` so you need to create that file or reference the `pages/enterprise/log_view/log_view_widget` in the toc.

File: pages/enterprise/log_view_widget.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+Of course, the Log View widget will provide you a way to sleuth around your log events, to 
+accomplish such actions as:
+
+* recording faults to diagnose and debug
+* identifying security breaches and other system and network misuses.
+* auditing
+
+When you build aggregations in the Log View widget expect it to help you to create highly 
+customizable reports and infographics. Furthermore, you can add them to your dashboards. 
+Also, you can save and retrieve them, in the the event you need to review that data again.
```
Comment (dennisoelkers) [inline pages/enterprise/log_view_widget.rst]:
```suggestion
Also, you can save and retrieve them, in the event you need to review that data again. 
```

File: pages/enterprise/log_view_widget.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+Log View Widget
+###############
+
+********
+Overview
+********
+
+Log View is a widget that presents your log data in a format similar to Common Log Format. 
+In other terms, it has the look and feel of console output. In addition, the Log View widget 
+allows you to scroll through log events and as it populates new lines in real-time.
```
Comment (dennisoelkers) [inline pages/enterprise/log_view_widget.rst]:
```suggestion
allows you to scroll through log events as it populates new lines in real-time.
```

File: pages/enterprise/log_view_widget.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+Of course, the Log View widget will provide you a way to sleuth around your log events, to
```
Comment (dennisoelkers) [inline pages/enterprise/log_view_widget.rst]: As a non-native speaker, I had to look up the meaning of `sleuth`. Maybe there is an easier word?

File: pages/searching/widgets.rst [hunk 4]
```diff
@@ -177,5 +183,8 @@ Widgets can be freely placed inside the search result grid. You can drag and dro
 left to the widget name or you resize them by using the gray arrow in their bottom-right corner.
 To expand a widget to the full grid width, click on the arrow in its top-right corner.
 
-.. image:: /images/searching/widget_repositioning_and_resizing.png
+.. image:: /images/searching/widget_repositioning_resizing.png
    :align: center
+
+If you want to expand the view of aggregated data in your *Log View* widget, go to :ref:`widget_focus` to 
+perfom those steps.
```
Comment (dennisoelkers) [inline pages/searching/widgets.rst]:
```suggestion
perform those steps.
```

**Review by dennisoelkers — CHANGES_REQUESTED** [review]
Thanks! Just added some minor corrections and one question.

File: pages/enterprise/log_view_widget.rst (hunk 1 above)
Comment (dulanism) [reply pages/enterprise/log_view_widget.rst]: Fixed

File: pages/enterprise/log_view_widget.rst (hunk 2 above)
Comment (dulanism) [reply pages/enterprise/log_view_widget.rst]: Updated. Thx

File: pages/enterprise/log_view_widget.rst (hunk 3 above)
Comment (dulanism) [reply pages/enterprise/log_view_widget.rst]: Changing "sleuth around" to "investigate".

File: pages/searching/widgets.rst (hunk 4 above)
Comment (dulanism) [reply pages/searching/widgets.rst]: Fixed. Thank you @dennisoelkers

**Review by dennisoelkers — APPROVED** [review]
Thanks a lot!

**Comment by dulanism** [comment]
I think I caught everything @dennisoelkers. Thanks again for the notes on improvement.

**Pull request merged**